index.html

```diff
@@ -309,6 +308,12 @@ <h3>Amol Agarwal</h3>
     window.addEventListener('DOMContentLoaded', (event) => {
       fetchDataFromFirebase();
     });
+    var c = [["#fc3a3a", "#eb6363"], ["#5562d9", "#959de8"], ["#facc00", "#ffe77a"], ["#ff96ff", "#ffbdff"]];
```

Comment:
This is great. Just add some spacing and comments. So that in future someone wants to can edit this.